include/sound/hdaudio_ext.h

```diff
@@ -84,6 +84,8 @@ int snd_hdac_ext_stream_init_all(struct hdac_bus *bus, int start_idx,
 		int num_stream, int dir);
 void snd_hdac_stream_free_all(struct hdac_bus *bus);
 void snd_hdac_link_free_all(struct hdac_bus *bus);
+struct hdac_ext_stream *hdac_ext_host_stream_compr_assign(struct hdac_bus *bus,
+				struct snd_compr_stream *cstream);
```

Comment:
is there any semantic reason behind the lack of 'snd' prefix? intended change or omission?

Reply:
Omission, thanks for reporting. Fixed in v6.